analyzer-core/src/test/java/ffb/analyzer/core/pubsub/PubSubTests.java

```diff
@@ -0,0 +1,63 @@
+package ffb.analyzer.core.pubsub;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public  class PubSubTests {
+    Publisher publisher = Publisher.getPublisher();
+    ExampleSubscriber subscriber = new ExampleSubscriber();
+
+    public void subscribeToExampleEvent() {
```

Comment:
`private`? Or is there a reason we need to have this `public`?

Reply:
It's public here because junit requires it. However, I updated this to be a generic postTestCleanup method and extracted the unsubscribeAll method to its own class where it has an access modifier of protected. I don't think we should limit it beyond that, as I suspect future tests in the pubsub model could leverage it.

Reply:
Whoops, was talking about the wrong method initially, but I think the logic applies to both unsubscribeAll and subscriveToExampleEvent. So I've moved them both to the package-specific Utils file where they're protected rather than public.